$NetBSD: patch-src_Makefile,v 1.1.2.2 2015/10/06 17:44:21 spz Exp $

Do not rely on bash being in /bin, especially when sh will do just fin...
Remove needless use of -v with mkdir; It's not POSIX and NetBSD doesn't have it.


--- src/Makefile.orig	2015-06-21 16:20:11.000000000 +0200
+++ src/Makefile	2015-10-02 05:40:26.000000000 +0200
@@ -150,7 +150,7 @@
 
 # ******************************************************************************
 # Initialisations
-SHELL = /bin/bash
+SHELL = /bin/sh
 
 .SUFFIXES:
 .SUFFIXES: .c .cpp .o .so
@@ -252,7 +252,7 @@
 	@$(LIBTOOL) --mode=link $(LINK.cc) -o $@ $(LIBRARY) $@.o -rpath $(libdir)
 
 $(EXIV2BIN): lib $(EXIV2OBJ) $(EXIV2COBJ)
-	mkdir -pv ../bin 2>&1 > /dev/null
+	mkdir -p ../bin 2>&1 > /dev/null
 	@$(LIBTOOL) --mode=link $(LINK.cc) -o ../bin/$@ $(LIBRARY) $(EXIV2OBJ) $(EXIV2COBJ) -rpath $(libdir)
 
 install-header:
